homeassistant/components/home_connect/sensor.py

```diff
@@ -70,16 +83,40 @@ async def async_update(self):
                     self._state = None
                 else:
                     seconds = self._sign * float(status[self._key][ATTR_VALUE])
-                    self._state = (
-                        dt_util.utcnow() + timedelta(seconds=seconds)
-                    ).isoformat()
+                    result_time = dt_util.utc_from_timestamp(
```

Comment:
Let's do this fix in a separate PR. We shouldn't mix feature additions and fixes in the same PR.